tests/test.sh

```diff
@@ -781,6 +795,8 @@ zero_local_jobs_test()
     remoteice2_pid=$!
     echo $remoteice2_pid > "$testdir"/remoteice2.pid
 
+    wait_for_proc_sleep 10 $localice_pid $remoteice1_pid $remoteice2_pid
+
```

Comment:
Once the daemons start, 'ps' reports R+ states for these processes.
```
R    running or runnable (on run queue)
```
```
+    is in the foreground process group
```
The daemons seem to respond once they have reached the 'S' state.
```
S    interruptible sleep (waiting for an event to complete)
```